experimental/DoubleAndHyperComplexes/src/Objects/induced_ENC.jl

```diff
@@ -22,8 +22,8 @@ function (fac::InducedENCChainFactory)(self::AbsHyperComplex, ind::Tuple)
   k + i - 1 > ngens(I0) && return sub(enc[i], elem_type(enc[i])[])[1]
   amb_ext_power = _exterior_power(enc, i)
   new_gens = elem_type(amb_ext_power)[]
-  for a in OrderedMultiIndexSet(k+i-1, ngens(I0))
-    push!(new_gens, wedge([repres(gen(I0, i)) for i in indices(a)]; parent=amb_ext_power))
+  for a in combinations(ngens(I0), k+i-1)
+    push!(new_gens, wedge([repres(gen(I0, i)) for i in a]; parent=amb_ext_power))
```

Comment:
```suggestion
  for as in combinations(gens(I0), k+i-1)
    push!(new_gens, wedge([repres(a) for a in as]; parent=amb_ext_power))
```
shouldn't this work as well here and saves some allocations?

Reply:
thanks for all of these changes, I guess I just sort of naively changed a lot of the `OrderedMultiSet` to `Combination` without paying attention to some of this added functionality we have (taking subsets or arbitrary objects instead of just `1:n`).